packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx

```diff
@@ -63,6 +66,24 @@ const CreateServiceWizard: React.FC<Props> = ({
     createResult?.build || null
   );
 
+  const { data: pluginsVersionData, refetch } = useQuery<{
+    plugins: Plugin[];
+  }>(GET_PLUGIN_VERSIONS_CATALOG, {
+    context: {
+      clientName: "pluginApiHttpLink",
+    },
+    variables: {
+      where: {
+        deprecated: {
+          equals: null,
+        },
+      },
+    },
+    onCompleted: () => {
+      if (!pluginsVersionData) refetch();
```

Comment:
Isnt it a potential loop?

Reply:
Why? @tupe12334

Reply:
I did it alreadt, didn't work.
Checking again why @EugeneTseitlin @tupe12334

Reply:
if the pluginsVersionData returns as null/undefined it will reflect again and again and again

Reply:
fixed